datafusion/physical-expr/benches/regexp.rs

```diff
@@ -78,20 +74,6 @@ fn flags(rng: &mut ThreadRng) -> StringArray {
 }
 
 fn criterion_benchmark(c: &mut Criterion) {
```

Comment:
I will file another PR to finish port regx_replace and delete this bench later today